Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -190,10 +191,21 @@ public bool Logout()
         /// </summary>
         /// <returns>Success or failure</returns>
         public bool LogoutV2()
+        {
+            Task<bool> task = Task.Run<bool>(async () => await LogoutV2Async());
+            return task.Result;
+        }
+
+        /// <summary>
+        /// Logout the user using the V2 endpoint asynchronous
+        /// </summary>
+        /// <returns>Success or failure</returns>
+        public async Task<bool> LogoutV2Async()
```

Comment:
Should we just call this `LogoutAsync`? I feel like we should remove both Logout and LogoutV2 for 6.0 in the future and then just have the single LogoutAsync method.